dss/config.py

```diff
@@ -230,21 +253,37 @@ def get_notification_email() -> str:
 
 
 class Replica(Enum):
```

Comment:
It might be worth exploring the possibility of adding `namedtuple(…, 'bucket_getter storage_schema blobstore_class hcablobstore_class') as a baseclass. It gives you immutable attributes and reduces the boilerplate. I just don't know if namedtuple is compatible with Enum.